### DISCUSSION TITLE
Serial unter DTU Settings?

### DISCUSSION THREAD

**Discussion by CrashCommando** [tbnobody/OpenDTU#331] [Q&A] [answered]
Hallo zusammen,

zu aller erst, ganz tolles Projekt! 
Hab bei mir auf Anhieb funktioniert. Die Ersparnis zur original Hoymiles DTU ist enorm. Dankeschön! 

Vielleicht eine blöde Frage, aber wozu ist das Feld "Serial" unter DTU Settings? Die Inverter Seriennummer wird ja bereits unter Inverter Settings eingetragen.

VG.

**Comment by tbnobody** [comment, marked as answer]
Die Seriennummern sind die "Adressen" der Geräte. Ganz grob sowas wie die IP Adressen im Netzwerk Umfeld. Die DTU Sendet an die Adresse (also Seriennummer) vom Inverter eine Anfrage. Der Inverter schickt die Antwort an die Adresse (also Seriennummer) von der DTU. 
Wie gesagt, das beschreibts nur grob...
Die automatisch generierte DTU Seriennummer sollte auch nur bearbeitet werden wenn man weiß was man tut :)

**Reply by stefan123t** [reply]
Damit kann man kurz gesagt sicherstellen dass sich mehrere DTUs aufgrund der gleichen DTU ID nicht über Funk mit den WRen logisch in die Quere kommen.
